server/lib/adapters/postgres.js

```diff
@@ -118,7 +121,6 @@ export function runQuery(client, query, options = {}) {
       // Run the query.
       currentQuery = client.query(query, (queryError, result) => {
         if (timer) clearTimeout(timer);
-        client.end();
```

Comment:
Are we 100% sure that this does not introduce an issue with hanging connections here ?